pkg/cmd/triggerbinding/list_test.go

```diff
@@ -91,6 +97,12 @@ func TestListTriggerBinding(t *testing.T) {
 			args:      []string{"list", "-n", "foo", "-o", "jsonpath={range .items[*]}{.metadata.name}{\"\\n\"}{end}"},
 			wantError: false,
 		},
+		{
+			name:      "TriggerBindings from all namespaces",
+			command:   command(t, tbs, now, ns),
+			args:      []string{"list", "--all-namespaces"},
+			wantError: false,
+		},
```

Comment:
Could you please also add a test for when no triggerbindings are available but `--all-namespaces` is used?

Reply:
Sure.

Reply:
I've added new test PTAL.

Reply:
Thanks!